words/scales.txt

```diff
@@ -118,8 +118,9 @@ hen
 rooster
 quail
 grouse
-
-# Musical scales
```

Comment:
Why deleting the comment/grouping?

Reply:
We don't use grouping anywhere else or order things in general. So between ordering them and grouping for consistency, and just removing the one grouping label, I chose to do the latter.

Plus, the mystic of why they are there is more fun 😛